### PR TITLE
fix errors

### PR DESCRIPTION
**Pull request by Nlopez4**

### CODE REVIEW

File: config/database.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-// const db = mongoose.connection
-// db.once('open', _ => {
-//   console.log('Database connected:', process.env.DATABASE_URL)
-// })
-
-// db.on('error', err => {
-//   console.error('connection error:', err)
-// })
+db.on('error', err => {
```
Comment (Nlopez4) [inline config/database.js:15]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.on('error', (err) => {
```

Comment (Nlopez4) [inline config/database.js:15]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.on("error", err => {
```

File: models/workoutlog.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// data for exercise, duration, and reps
+
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const exerSchema = new mongoose.Schema({
+    type: String,
+    enum: ['Pushup', 'Pull Ups', 'Chin Ups', 'Barbell Squats', 'Air Squats', 'Bulgarian Split Squat', 'Goblet Squat','Dumbbell Squat', 'Wall Sits', 'Conventional Deadlift', 'Sumo Deadlift', 'Kettle Bells', 'Outdoor Walk', 'Indoor Walk', 'Outdoor Run', 'Treadmill', 'Row', 'Bicycle', 'Bench Press', 'Dips', 'Dumbbell Press', 'Dips']
```
Comment (Nlopez4) [inline models/workoutlog.js:9]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
    enum: ["Pushup", 'Pull Ups', 'Chin Ups', 'Barbell Squats', 'Air Squats', 'Bulgarian Split Squat', 'Goblet Squat','Dumbbell Squat', 'Wall Sits', 'Conventional Deadlift', 'Sumo Deadlift', 'Kettle Bells', 'Outdoor Walk', 'Indoor Walk', 'Outdoor Run', 'Treadmill', 'Row', 'Bicycle', 'Bench Press', 'Dips', 'Dumbbell Press', 'Dips']
```

File: config/database.js [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+  console.error('connection error:', err)
```
Comment (Nlopez4) [inline config/database.js:16]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
  console.error("connection error:", err)
```

File: config/database.js [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+});
 
-// mongoose.connect(process.env.DATABASE_URL, {
-//     useNewUrlParser: true, 
-//     useCreateIndex: true,
-//     useUnifiedTopology: true 
-// });
+const db = mongoose.connection
+db.once('open', _ => {
+  console.log('Database connected:', process.env.DATABASE_URL)
```
Comment (Nlopez4) [inline config/database.js:12]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
  console.log("Database connected:", process.env.DATABASE_URL)
```

File: config/database.js [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
+    useUnifiedTopology: true 
+});
 
-// mongoose.connect(process.env.DATABASE_URL, {
-//     useNewUrlParser: true, 
-//     useCreateIndex: true,
-//     useUnifiedTopology: true 
-// });
+const db = mongoose.connection
+db.once('open', _ => {
```
Comment (Nlopez4) [inline config/database.js:11]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.once("open", _ => {
```

File: server.js [hunk 6]
```diff
@@ -35,22 +36,23 @@ require("./config/passport");
...
 app.use('/', router);
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
   app.use(passport.initialize());
   app.use(passport.session());
-
+  
+// workout log 
+app.use('/', excerRoute);
```
Comment (Nlopez4) [inline server.js:50]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
app.use("/", excerRoute); 
```

File: controllers/workoutlog.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+const log = require('../models/workoutlog')
+
+module.exports = {
+  index,
+};
+
+function index(req, res) {
+  res.render('workoutlog/index')
+};
```
Comment (Nlopez4) [inline controllers/workoutlog.js:9]: Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
} 
```

File: config/database.js [hunk 8]
```diff
@@ -1,17 +1,17 @@
+// mongoose connection
+const mongoose = require('mongoose');
```
Comment (Nlopez4) [inline config/database.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const mongoose = require("mongoose");
```

File: models/workoutlog.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+});
+
+const repSchema = new mongoose.Schema({
+   reps: {type: Number, min: 1, max: 100, required: true}
+}); 
+
+
+
+module.exports = mongoose.model('exercise', exerSchema);
+module.exports = mongoose.model('duration', durSchema);
```
Comment (Nlopez4) [inline models/workoutlog.js:23]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
module.exports = mongoose.model("duration", durSchema);
```

File: server.js [hunk 10]
```diff
@@ -1,16 +1,17 @@
 require("dotenv").config();
 /* ==== External Modules ==== */
 const express = require("express");
-//const morgan = require('morgan');
+const morgan = require('morgan');
 const session = require("express-session");
 const passport = require("passport");
 const indexRoutes = require('./routes/index');
 const userRoutes = require('./routes/user');
+const excerRoute = require('./routes/workoutlog');
```
Comment (Nlopez4) [inline server.js:9]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const excerRoute = require("./routes/workoutlog"); 
```

File: controllers/workoutlog.js [hunk 11]
```diff
@@ -0,0 +1,10 @@
+const log = require('../models/workoutlog')
```
Comment (Nlopez4) [inline controllers/workoutlog.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const log = require("../models/workoutlog")
```

File: server.js [hunk 12]
```diff
@@ -1,16 +1,17 @@
 require("dotenv").config();
 /* ==== External Modules ==== */
 const express = require("express");
-//const morgan = require('morgan');
+const morgan = require('morgan');
```
Comment (Nlopez4) [inline server.js:4]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const morgan = require("morgan");
```

File: server.js [hunk 13]
```diff
@@ -1,16 +1,17 @@
...
 const session = require("express-session");
 const passport = require("passport");
 const indexRoutes = require('./routes/index');
 const userRoutes = require('./routes/user');
+const excerRoute = require('./routes/workoutlog'); 
+const workMod = require('./models/workoutlog');
```
Comment (Nlopez4) [inline server.js:10]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const workMod = require("./models/workoutlog");
```

File: routes/workoutlog.js [hunk 14]
```diff
@@ -0,0 +1,12 @@
+// For workoutlog data
+const express = require('express');
```
Comment (Nlopez4) [inline routes/workoutlog.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const express = require("express"); 
```

File: controllers/workoutlog.js [hunk 15]
```diff
@@ -0,0 +1,13 @@
...
+
+
+
+module.exports = {
+  index,
+};
+
+function index(req, res) {
+  res.send('workoutlog/index')
+};
```
Comment (Nlopez4) [inline controllers/workoutlog.js:12]: Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
} 
```

File: server.js (hunk 12 above)
Comment (Nlopez4) [inline server.js:4]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const morgan = require("morgan");
```

File: models/workoutlog.js [hunk 16]
```diff
@@ -0,0 +1,30 @@
+// data for exercise
+
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const exerSchema = new mongoose.Schema({
+    exercise: {
+        type: String,
+        enum: ['Pushup', 'Pull Ups', 'Chin Ups', 'Barbell Squats', 'Air Squats', 'Bulgarian Split Squat', 'Goblet Squat','Dumbbell Squat', 'Wall Sits', 'Conventional Deadlift', 'Sumo Deadlift', 'Kettle Bells', 'Outdoor Walk', 'Indoor Walk', 'Outdoor Run', 'Treadmill', 'Row', 'Bicycle', 'Bench Press', 'Dips', 'Dumbbell Press', 'Dips']
```
Comment (Nlopez4) [inline models/workoutlog.js:10]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
    enum: ["Pushup", 'Pull Ups', 'Chin Ups', 'Barbell Squats', 'Air Squats', 'Bulgarian Split Squat', 'Goblet Squat','Dumbbell Squat', 'Wall Sits', 'Conventional Deadlift', 'Sumo Deadlift', 'Kettle Bells', 'Outdoor Walk', 'Indoor Walk', 'Outdoor Run', 'Treadmill', 'Row', 'Bicycle', 'Bench Press', 'Dips', 'Dumbbell Press', 'Dips']
```

File: config/database.js (hunk 1 above)
Comment (Nlopez4) [inline config/database.js:15]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.on('error', (err) => {
```

File: config/database.js (hunk 3 above)
Comment (Nlopez4) [inline config/database.js:16]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
  console.error("connection error:", err)
```

File: controllers/workoutlog.js [hunk 17]
```diff
@@ -0,0 +1,13 @@
...
+
+
+
+
+module.exports = {
+  index,
+};
+
+function index(req, res) {
+  res.send('workoutlog/index')
```
Comment (Nlopez4) [inline controllers/workoutlog.js:11]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
  res.send("workoutlog/index")
```

File: config/database.js (hunk 4 above)
Comment (Nlopez4) [inline config/database.js:12]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
  console.log("Database connected:", process.env.DATABASE_URL)
```

File: routes/workoutlog.js [hunk 18]
```diff
@@ -0,0 +1,12 @@
+// For workoutlog data
+const express = require('express'); 
+const router = express.Router();
+const workoutCtrl = require('../controllers/workoutlog');
```
Comment (Nlopez4) [inline routes/workoutlog.js:4]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const workoutCtrl = require("../controllers/workoutlog");
```

File: models/workoutlog.js [hunk 19]
```diff
@@ -0,0 +1,30 @@
...
+
+const repSchema = new mongoose.Schema({
+   reps: {type: Number, min: 1, max: 100, required: true}
+}); 
+
+
+
+const findExercise = module.exports = mongoose.model('exercise', exerSchema);
+const findDuration = module.exports = mongoose.model('duration', durSchema);
+const findReps = module.exports = mongoose.model('reps', repSchema);
```
Comment (Nlopez4) [inline models/workoutlog.js:26]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const findReps = module.exports = mongoose.model("reps", repSchema);
```

File: server.js (hunk 6 above)
Comment (Nlopez4) [inline server.js:50]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
app.use("/", excerRoute); 
```

File: config/database.js (hunk 5 above)
Comment (Nlopez4) [inline config/database.js:11]: Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.once('open', (_) => {
```

File: config/database.js (hunk 1 above)
Comment (Nlopez4) [inline config/database.js:15]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.on("error", err => {
```

File: controllers/workoutlog.js [hunk 20]
```diff
@@ -0,0 +1,13 @@
+const log = require('../models/workoutlog')
```
Comment (Nlopez4) [inline controllers/workoutlog.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const log = require("../models/workoutlog")
```

File: models/workoutlog.js [hunk 21]
```diff
@@ -0,0 +1,30 @@
...
+    duration: {type: Number, min: 1, max: 60, required: true}
+});
+
+const repSchema = new mongoose.Schema({
+   reps: {type: Number, min: 1, max: 100, required: true}
+}); 
+
+
+
+const findExercise = module.exports = mongoose.model('exercise', exerSchema);
```
Comment (Nlopez4) [inline models/workoutlog.js:24]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const findExercise = module.exports = mongoose.model("exercise", exerSchema);
```

File: server.js (hunk 10 above)
Comment (Nlopez4) [inline server.js:9]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const excerRoute = require("./routes/workoutlog"); 
```

File: routes/workoutlog.js (hunk 14 above)
Comment (Nlopez4) [inline routes/workoutlog.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const express = require("express"); 
```

File: models/workoutlog.js [hunk 22]
```diff
@@ -0,0 +1,30 @@
...
+const findDuration = module.exports = mongoose.model('duration', durSchema);
```
Comment (Nlopez4) [inline models/workoutlog.js:25]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const findDuration = module.exports = mongoose.model("duration", durSchema);
```

File: config/database.js (hunk 8 above)
Comment (Nlopez4) [inline config/database.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const mongoose = require("mongoose");
```

File: server.js [hunk 23]
```diff
@@ -1,18 +1,22 @@
...
 
-//const methodOverride = require("method-override");
 
-/* ==== Internal Modules ==== */
 
+/* ==== Internal Modules ==== */
+const indexRoutes = require('./routes/index');
+const userRoutes = require('./routes/user');
+const excerRoute = require('./routes/workoutlog'); 
+const durationRoute = require('./routes/duration');
```
Comment (Nlopez4) [inline server.js:17]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const durationRoute = require("./routes/duration");
```

File: models/reps.js [hunk 24]
```diff
@@ -0,0 +1,18 @@
...
+ }); 
+
+ const findReps = module.exports = mongoose.model('reps', repSchema);
+
+
+ // display 
+ findReps.
+    find().
+    limit(100).
+    select('reps');
```
Comment (Nlopez4) [inline models/reps.js:18]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
    select("reps"); 
```

File: routes/workoutlog.js [hunk 25]
```diff
@@ -0,0 +1,19 @@
+// For workoutlog data
+const express = require('express');
```
Comment (Nlopez4) [inline routes/workoutlog.js:2]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const express = require("express"); 
```

File: models/workoutlog.js [hunk 26]
```diff
@@ -0,0 +1,20 @@
...
+        enum: ['Pushup', 'Pull Ups', 'Chin Ups', 'Barbell Squats', 'Air Squats', 'Bulgarian Split Squat', 'Goblet Squat','Dumbbell Squat', 'Wall Sits', 'Conventional Deadlift', 'Sumo Deadlift', 'Kettle Bells', 'Outdoor Walk', 'Indoor Walk', 'Outdoor Run', 'Treadmill', 'Row', 'Bicycle', 'Bench Press', 'Dips', 'Dumbbell Press', 'Dips']
+    }
+});
+
+const findExercise = module.exports = mongoose.model('exercise', exerSchema);
+
+
+// display 
+findExercise.find({}, function (err, result) {
+    if (err) return handleError(err);
```
Comment (Nlopez4) [inline models/workoutlog.js:19]: Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
    if (err) {return handleError(err);}
```

File: controllers/workoutlog.js [hunk 27]
```diff
@@ -0,0 +1,10 @@
+const log = require('../models/workoutlog'); 
+
+module.exports = {
+  index,
+};
+
+function index(req, res) {
+  res.send('workoutlog/index')
+};
```
Comment (Nlopez4) [inline controllers/workoutlog.js:9]: Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
} 
```

File: config/database.js [hunk 28]
```diff
@@ -1,17 +1,17 @@
...
 
-// const db = mongoose.connection
-// db.once('open', _ => {
-//   console.log('Database connected:', process.env.DATABASE_URL)
-// })
-
-// db.on('error', err => {
-//   console.error('connection error:', err)
-// })
+db.on('error', err => {
```
Comment (Nlopez4) [inline config/database.js:15]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
db.on("error", err => {
```

File: server.js [hunk 29]
```diff
@@ -1,18 +1,22 @@
...
 
 
-//const methodOverride = require("method-override");
 
-/* ==== Internal Modules ==== */
 
+/* ==== Internal Modules ==== */
+const indexRoutes = require('./routes/index');
+const userRoutes = require('./routes/user');
+const excerRoute = require('./routes/workoutlog');
```
Comment (Nlopez4) [inline server.js:16]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const excerRoute = require("./routes/workoutlog"); 
```

File: server.js [hunk 30]
```diff
@@ -35,22 +39,31 @@ require("./config/passport");
...
 app.use('/', router);
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
   app.use(passport.initialize());
   app.use(passport.session());
-
+  
+// workout log 
+app.use('/', excerRoute);
```
Comment (Nlopez4) [inline server.js:61]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
app.use("/", excerRoute); 
```

File: models/duration.js [hunk 31]
```diff
@@ -0,0 +1,18 @@
...
+
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const durSchema = new mongoose.Schema({
+    duration: {type: Number, min: 1, max: 60, required: true}
+});
+
+const findDuration = module.exports = mongoose.model('duration', durSchema);
```
Comment (Nlopez4) [inline models/duration.js:11]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const findDuration = module.exports = mongoose.model("duration", durSchema);
```

File: controllers/workoutlog.js [hunk 32]
```diff
@@ -0,0 +1,10 @@
+const log = require('../models/workoutlog'); 
+
+module.exports = {
+  index,
+};
+
+function index(req, res) {
+  res.send('workoutlog/index')
```
Comment (Nlopez4) [inline controllers/workoutlog.js:8]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
  res.send("workoutlog/index")
```

File: routes/reps.js [hunk 33]
```diff
@@ -0,0 +1,20 @@
+// For workoutlog data
+const express = require('express'); 
+const router = express.Router();
+const repsCtrl = require('../controllers/reps');
```
Comment (Nlopez4) [inline routes/reps.js:4]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const repsCtrl = require("../controllers/reps");
```

File: controllers/duration.js [hunk 34]
```diff
@@ -0,0 +1,9 @@
+const log = require('../models/duration');
```
Comment (Nlopez4) [inline controllers/duration.js:1]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const log = require("../models/duration");
```

File: controllers/reps.js [hunk 35]
```diff
@@ -0,0 +1,9 @@
+const log = require('../models/reps');
+
+module.exports = {
+  index,
+};
+
+function index(req, res) {
+  res.send('reps/index')
```
Comment (Nlopez4) [inline controllers/reps.js:8]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
  res.send("reps/index")
```

File: models/duration.js [hunk 36]
```diff
@@ -0,0 +1,18 @@
...
+
+
+// display 
+findDuration.
+    find().
+    limit(60).
+    select('reps');
```
Comment (Nlopez4) [inline models/duration.js:18]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
    select("reps"); 
```

File: models/workoutlog.js [hunk 37]
```diff
@@ -0,0 +1,20 @@
+// data for exercise
+
+const mongoose = require('mongoose');
```
Comment (Nlopez4) [inline models/workoutlog.js:3]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7827980)
```suggestion
const mongoose = require("mongoose");
```

**Review by MaricelaDuron0809 — APPROVED** [review]

**Pull request merged**